synced-files/.github/labels.yml

```diff
@@ -0,0 +1,139 @@
+# This is an auto-generated file. Please don't edit this file directly.
+
+# This file is automatically generated by appending two files:
+# 1. The Exercism-wide labels: defined in https://github.com/exercism/org-wide-files/blob/main/synced-files/.github/labels.yml)
+# 2. The repository-specific labels: defined in a `.appends/.github/labels.yml` file (optional)
+#
+# If you want to use additional labels besides the Exercism-wide labels,
+# don't edit this file directly but instead create a `.appends/.github/labels.yml`
+# file within this repo.
+#
+# Any changes to the repo's `.appends/.github/labels.yml`
+# file or the Exercism-wide `labels.yml` file will automatically result in this
+# file being regenerated and the labels being updated.
```

Comment:
```suggestion
# Any changes to the repo's `.appends/.github/labels.yml`
# file or the Exercism-wide `labels.yml` file will automatically result in this
# file being regenerated and the labels being updated. This typically takes
# 5-10 minutes.
```

Reply:
What line-widths are we aiming for here? Feels like we could definitely compress this into at least 3 lines :)

Reply:
I have no idea, I tried to follow Erik's line width :D

Reply:
:D @ErikSchierboom? :)

Reply:
I've updated your suggestion to spread it onto three lines :)

Reply:
No particular line width